File: src/main/java/io/bloco/faker/helpers/MathHelper.java
package io.bloco.faker.helpers;

public class MathHelper {

    public MathHelper() {
    }

    public double round(double number, int precision) {
        double precisionPow = Math.pow(10, precision);
        return Math.round(number * precisionPow) / precisionPow;
    }
}

File: lib/src/main/kotlin/io/bloco/faker/helpers/MathHelper.kt
package io.bloco.faker.helpers

import kotlin.math.pow
import kotlin.math.roundToInt

class MathHelper {
    fun round(number: Double, precision: Int): Double {
        val precisionPow = 10.0.pow(precision.toDouble())
        return (number * precisionPow).roundToInt() / precisionPow
    }
}